options/locale/locale_en-US.ini

```diff
@@ -1710,7 +1710,7 @@ pulls.delete.text = Do you really want to delete this pull request? (This will p
 
 milestones.new = New Milestone
 milestones.closed = Closed %s
-milestones.updated = Updated
+milestones.updated = Updated %s
```

Comment:
The key could be `updated_on`

Reply:
Originally it was `milestones.update_ago`

Reply:
To keep consistent with `closed`, so `updated` is also accepted?

Reply:
A clear key could help translators a lot.

If the key is `updated_on`, the translators could know it's for date/time

But if the key is `updated`, the translators have to guess: update by someone? updated to something? updated on someday?

I really prefer to make every name exactly clear.

Reply:
At least, please revert to the old `update_ago` if the text doesn't change, then translators do not need to do unnecessary re-translating.